File: FEBRUARY/21-02-2026/762.cpp
class Solution {
public:
    int countPrimeSetBits(int left, int right) {
        int count = 0;

        int isPrime[33] = {
            0, 0, 2, 3, 0, 5, 0, 7, 0, 0, 0, 11, 0,
            13, 0, 0, 0, 17, 0, 19, 0, 0, 0, 23,
            0, 0, 0, 0, 0, 29, 0, 31, 0
        };

        while(left <= right) {
            if(isPrime[__builtin_popcount(left)]) {
                count++;
            }
            left++;
        }

        return count;
    }
};

File: FEBRUARY/21-02-2026/762.py
class Solution:
    def countPrimeSetBits(self, left: int, right: int) -> int:
        setBits = {2,3,5,7,11,13,17,19}
        count = 0 
        
        for i in range(left, right + 1): 
            onecount = i.bit_count()

            if onecount in setBits: 
                count += 1 
        
        return count